src/commands/edge_app.rs

```diff
@@ -95,6 +95,52 @@ impl EdgeAppCommand {
         Ok(())
     }
 
+    pub fn create_in_place(&self, name: &str, path: &Path) -> Result<(), CommandError> {
```

Comment:
I don't want to do this in create because there will literally be if-else everywhere

Reply:
that's fine but there is a lot of common code, no?

Reply:
I think these functions have 50% of the same code. But anyhow It will be ugly.